src/frontend/flight/sync/writer.rs

```diff
@@ -838,12 +838,13 @@ mod tests {
                 sync_mgr.flush_syncs().await.unwrap();
 
                 for (o, durs) in durable_sequences.iter_mut().enumerate() {
-                    let origin = o as i64;
+		    // What is this supposed to mean?
+                    let origin = *o;
```

Comment:
I am quite confused about what was going on here. The index in the sequence was being used as the origin id?

Reply:
Yeah, it looks like this is just for convenience for this test case (`mut durable_sequences` is parameterized and in the test data the origins have consecutive IDs).

The `to_string()` below makes sense to me.